Remove pkg and update readme (#46)
Sage-Bionetworks/rocc-db-client: README.md

## Usage

Install the dependencies.

    npm ci

Build the client.

    npm run build

> If this happens, the next step (running the client), will not work.  If you

Reviewer: ```suggestion
```

> are interested in seeding the database, refer to [Using node to Seed the DB].

Reviewer: ```suggestion
```

Run the client.

    node ./dist/src/index.js

For example:

```console
$ node ./dist/src/index.js help
Usage: rocc-db-client [global options] command

Client for managing a ROCC DB instance.

Options:
  -v, --version          output the current version
  -d, --debug            output extra debugging
  --uri <uri>            MongoDB uri (default: "mongodb://localhost:27017/rocc")
  --username <username>  MongoDB username (default: "roccmongo")
  --password <password>  MongoDB password (default: "roccmongo")
  -h, --help             display help for command

Commands:
  ping                   ping the MongoDB instance
  remove-collections     remove all collections
  seed <directory>       empty and seed the db with the JSON files from the directory specified
  help [command]         display help for command
```

### Using node to Seed the DB

> If your local machine was unable to package the binary files with `npm run

Reviewer: ```suggestion
```

> package`, you can alternatively seed the database with:

Reviewer: ```suggestion
You can alternatively seed the database directly with node:
```
